.github/workflows/vale.yml

```
@@ -0,0 +1,34 @@
+name: Validate content with Vale
+
+on:
+  pull_request:
+    paths:
+      - 'content/**'
+
+  # Allows manual workflow run (must in default branch to work)
+  workflow_dispatch:
+
+jobs:
+  build:
```

```suggestion
  lint-content:
```
Non-blocking: `build` as a name for the job feels out of place in this context.

---

Shh, you'll expose the fact I don't know what I'm doing and just copy/pasta things to get them to work ;) 

Will fix.